lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -354,7 +366,36 @@ def send_bulk(data)
     begin
       response = client.bulk body: data
       if response['errors']
-        log.error "Could not push log to Elasticsearch: #{response}"
+        errors = Hash.new(0)
+        errors_bad = 0
+        errors_unrecognized = 0
+        # Count up the individual error types returned for each item
+        response['items'].each do |item|
+          begin
+            if [429, 500].include?(item['create']['status'])
```

Comment:
I'll check the elasticsearch-ruby API to see if it includes successful return values in the data set.